exonum/src/node/consensus.rs

```diff
@@ -578,16 +578,18 @@ impl NodeHandler {
             }
         };
 
-        if !tx.verify() {
-            return;
-        }
+        self.execute_later(InternalRequest::VerifyTx(tx));
+    }
 
+    /// Handles an already verified transaction.
+    pub fn handle_verified_tx(&mut self, tx: &dyn Transaction) {
         // We don't care about result, because situation when transaction received twice
```

Comment:
Do I understand this comment correctly that a signature for the same transaction may be verified multiple times if a transaction is received from multiple nodes? If so, this seems like an area for future improvement; we want to ignore known transactions right away.